---
bool success
string message
geometry_msgs/TransformStamped[] Tcm  # camera <- marker   transformation
geometry_msgs/TransformStamped[] Twe  # world  <- effector transformation
